plans/ant/plan.sh

```diff
@@ -0,0 +1,22 @@
+pkg_origin=core
+pkg_name=ant
+pkg_version=1.9.7
+pkg_maintainer='The Habitat Maintainers <humans@habitat.sh>'
+pkg_license=('Apache 2')
```

Comment:
can you change the license string to read `Apache-2.0`, per the [SPDX License List](http://spdx.org/licenses/)?